pkg/persistence/redis/pool.go

```diff
@@ -0,0 +1,157 @@
+package redis
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/goharbor/harbor-scanner-clair/pkg/etc"
+
+	"github.com/FZambia/sentinel"
+	"github.com/gomodule/redigo/redis"
+	log "github.com/sirupsen/logrus"
+)
+
+// GetPool constructs a redis.Pool for the given RedisPool config params.
+//
+// The currently supported Redis URL schemas correspond to a Redis server
+// run in standalone and Sentinel modes:
+// - redis://user:password@standalone_host:port/db-number
+// - redis+sentinel://user:password@sentinel_host1:port1,sentinel_host2:port2/master_name/db-number
```

Comment:
@danielpacak this is not master_name, it is actually a sentinel group name.

Reply:
Got it. Reverted it to `monitor-name`